### PR TITLE
Update AddressFormat class to ps standards

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | `develop` |
| Description? | Upgrading the PHPDoc comments of this class, using the latest PS standards and also applying the PS standards to some of the code. Also getting the Franglais out of the comments, getting rid of things the Symfony code sniffer nags about, etc. It might have some additional `todo` tags, depending on whether the code was comprehensible or not. |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | N/A |
| How to test? | N/A |

Warning: this PR also has a code cleanup. Please check this first before merging!

### CODE REVIEW

File: classes/AddressFormat.php [hunk 1]
```diff
@@ -528,76 +536,91 @@ public static function getFormattedLayoutData($address)
...
+     * @param int $idCountry Country ID
      *
      * @return false|null|string Address format
+     * @since 1.7.0
      */
-    protected function _getFormatDB($id_country)
+    protected function getFormatDB($idCountry)
     {
-        if (!Cache::isStored('AddressFormat::_getFormatDB'.$id_country)) {
+        if (!Cache::isStored('AddressFormat::_getFormatDB'.$idCountry)) {
```
Comment (mickaelandrieu) [inline classes/AddressFormat.php]:
```diff
- AddressFormat::_getFormatDB
+ AddressFormat::getFormatDB
```

File: classes/AddressFormat.php [hunk 2]
```diff
@@ -528,76 +536,91 @@ public static function getFormattedLayoutData($address)
...
             $format = Db::getInstance()->getValue('
 			SELECT format
 			FROM `'._DB_PREFIX_.$this->def['table'].'`
-			WHERE `id_country` = '.(int)$id_country);
+			WHERE `id_country` = '.(int) $idCountry);
             $format = trim($format);
-            Cache::store('AddressFormat::_getFormatDB'.$id_country, $format);
+            Cache::store('AddressFormat::_getFormatDB'.$idCountry, $format);
```
Comment (mickaelandrieu) [inline classes/AddressFormat.php]:
```diff
- AddressFormat::_getFormatDB
+ AddressFormat::getFormatDB
```

File: classes/AddressFormat.php [hunk 3]
```diff
@@ -528,76 +536,91 @@ public static function getFormattedLayoutData($address)
...
 
             return $format;
         }
 
-        return Cache::retrieve('AddressFormat::_getFormatDB'.$id_country);
+        return Cache::retrieve('AddressFormat::_getFormatDB'.$idCountry);
```
Comment (mickaelandrieu) [inline classes/AddressFormat.php]:
```diff
- AddressFormat::_getFormatDB
+ AddressFormat::getFormatDB
```

File: classes/AddressFormat.php (hunk 1 above)
Comment (firstred) [inline classes/AddressFormat.php]: :open_mouth: 
Well found, sir! :smile:

**Comment by aleeks** [comment]
Thank you!

**Pull request merged**